src/pages/Animal/MyPets/index.tsx

```diff
@@ -22,10 +24,12 @@ const MyPets = (): JSX.Element => {
   }, [navigation]);
 
   const fetchPets = (): void => {
-    const { currentUser } = auth();
-    const user = firestore().collection('users').doc(currentUser?.uid);
+    const currentUser = userAPI.currentUser();
+    const user = userAPI.userFirestoreDocument(currentUser?.uid);
 
-    firestore().collection('animals').orderBy('name').where('owner', '==', user)
+    animalAPI.animalFirestoreCollection()
+      .orderBy('name')
+      .where('owner', '==', user)
```

Comment:
I guess it could be very nice if it was `Animals.getOwnedBy(user)`

Reply:
Fixed by new commit.